packages/amazonq/src/app/inline/recommendationService.ts

```diff
@@ -30,6 +31,9 @@ export class RecommendationService {
             context,
         }
         const requestStartTime = Date.now()
+        TelemetryHelper.instance.setInvokeSuggestionStartTime()
+        TelemetryHelper.instance.setPreprocessEndTime()
+        TelemetryHelper.instance.setSdkApiCallStartTime()
 
         // Handle first request
         const firstResult: InlineCompletionListWithReferences = await languageClient.sendRequest(
```

Comment:
unrelated to work here, but do we need to encrypt this request?

Reply:
🤔 good question, I guess I probably should

Reply:
I'm asking internally for more clarification, but judging from the types it looks like inline can't be encrypted